core/rawdb/accessors_chain.go

```diff
@@ -469,6 +500,228 @@ func ReadTdRLP(db ethdb.Reader, hash common.Hash, number uint64) rlp.RawValue {
 	return nil // Can't find the data anywhere.
 }
 
+// ReadHeadsHashes retreive's the heads hashes of the blockchain.
+func ReadTermini(db ethdb.Reader, hash common.Hash) []common.Hash {
```

Comment:
It sounds to me like the "termini" returned by this function, is a collection of four hashes: 3 subordinate prior coincident block hashes and your own prior coincident block hash.

Please create a type for this, instead of leaving it as an arbitrary array of hashes. Perhaps:
```
type BlockTermini struct {
    priorDomCoincident common.Hash
    priorSubCoincidents []common.Hash
}
```

Reply:
After the discussion, let's table this until we have a nice view of the manifest and potentially termini will be subsumed into the manifest.

Reply:
https://github.com/dominant-strategies/go-quai/issues/429 link to the issue.